packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -244,13 +244,9 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
       context: context,
       builder: (context) => FilterDialog(
         controller: _networkController,
-        onApplyFilter: (query) => _networkController.filterData(
-          QueryFilter.parse(
-            query,
-            _networkController.filterArgs,
-          ),
-        ),
+        onApplyFilter: _networkController.filterData,
```

Comment:
should we remove onApplyFilter and just let the filterDialoag call 
controller.filterData?

Reply:
done